doc/api-reference/rest-api.rst

```diff
@@ -114,11 +114,11 @@ Response:
 
 A list of error lines extracted from the log. If the calculation was successfull, the list is empty.
 
-**********************************
-GET /v1/calc/:calc_id/aggrisk_tags
-**********************************
+********************************
+GET /v1/calc/:calc_id/get_impact
+********************************
```

Comment:
All other URLs are not verbs, so for consistency I would prefer `impact` over `get_impact`

Reply:
I was using `impact` at first, but it collided with `openquake.engine.impact`, so I need a different name. Do you have any suggestion?

Reply:
I renamed the endpoint to `impact`, internally calling a Django view called `impact_results`.